articles/aks/http-application-routing.md

```diff
@@ -101,7 +101,7 @@ kind: Ingress
 metadata:
   name: party-clippy
   annotations:
-    kubernetes.io/ingress.class: addon-http-application-routing
+    kubernetes.io/ingress.class: addon-http-application-routing-nginx-ingress
```

Comment:
Unfortunately this is not the case. The nginx-ingress controller is configured as follows
```
      --ingress-class=addon-http-application-routing
```